front/style/svelte-style.scss

```diff
@@ -0,0 +1,82 @@
+//Contains all styles not unique to a svelte component.
+
+//ConsoleGlobalMessage
+div.main-console.nes-container {
+  pointer-events: auto;
+  margin-left: auto;
+  margin-right: auto;
+  top: 20vh;
+  width: 50vw;
+  height: 50vh;
+  padding: 0;
+  background-color: #333333;
+
+  .btn-action{
+    margin: 10px;
+    text-align: center;
+  }
+
+  .main-global-message {
+    width: 100%;
+    max-height: 100%;
+  }
+
+  .main-global-message h2 {
+    text-align: center;
+    color: white;
+  }
+
+  div.global-message {
+    display: flex;
+    max-height: 100%;
+    width: 100%;
+  }
+
+  div.menu {
+    flex: auto;
+  }
+
+  div.menu button {
+    margin: 7px;
+  }
+
+  .main-input {
+    width: 95%;
+  }
+
+//InputTextGlobalMessage
+  .section-input-send-text {
+    margin: 10px;
+  }
+
+  .section-input-send-text .input-send-text .ql-editor{
+    color: white;
+    min-height: 200px;
+  }
+
+  .section-input-send-text .ql-toolbar{
+    background: white;
+  }
+
+//UploadAudioGlobalMessage
+  .section-input-send-audio {
+    margin: 10px;
+  }
+
+  .section-input-send-audio .input-send-audio {
+    text-align: center;
+  }
+
+  .section-input-send-audio #input-send-audio{
+    display: none;
+  }
+
+  .section-input-send-audio div.input-send-audio label{
+    color: white;
+  }
+
+  .section-input-send-audio div.input-send-audio img{
+    height: 150px;
+    cursor: url('./images/cursor_pointer.png'), pointer;
+  }
```

Comment:
This seems pretty unique to the audio component to me.
I'd probably put this in the component rather than in this file.